app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.where(token: params[:id]).first
+    if user
+      @token = user.token 
+    else
+      redirect_to expired_token_path unless user
```

Comment:
You don't need `unless user` here, as it can't reach this point `if user` (line 4).